src/kosha/app/search.clj

```diff
@@ -3,13 +3,14 @@
             [kosha.app.util :as util]))
 
 
-(defn get-ragam
+(defn get-ragams
   "Retrieves 10 best matches for given ragam name."
-  [{:keys [params] :as request}]
-  (let [ragams (db-search/ragams (:q params) 10)]
+  [query]
+  (let [ragams (db-search/ragams query 10)]
```

Comment:
If you already have a config, pull this `10` out into that. At the least, pull it out as a global var in this ns. See http://wiki.c2.com/?MagicNumber.